llvm/test/Transforms/Attributor/cb_liveness_enabled.ll

```diff
@@ -192,6 +192,7 @@ define i32 @test_ncheck2(i32 %0) #0 {
 
 attributes #0 = { noinline nounwind sspstrong uwtable}
 
+;.
```

Comment:
unrelated change?

Reply:
It's in a standalone commit. I can push it first. Look at commit #3 only.